luaui/configs/bar_hotkeys_grid.lua

```diff
@@ -1,6 +1,6 @@
--- BAR hotkey config file: ergokeys for grid menu	
+-- BAR hotkey config file: optimized for use with grid build menu.	
 	local bindings = {
-		{            "esc", "select", "AllMap++_ClearSelection+" },
+		{            "esc", "select", "AllMap++_ClearSelection_SelectNum_0+" },
```

Comment:
Why is this necessary?

Reply:
Yes as I recall I got some errors with it during testing (missing token?) so I thought maybe it was missing the answer to the question, "what do I select?" My thought was to tell it select nothing, and it seemed to fix the error.

Reply:
Gotcha, I'll keep that in mind, thank you @sprunk

Reply:
All clear from my end